### PR TITLE
added 2 workflows + updated one

### PR DESCRIPTION
**Pull request by radugalbenu**
## Description of changes (updated or new workflows)
1. Added workflow for update kubeconfig with AWS EKS cluster
2. Added workflow for exec into the shell of a pod
3. Modified workflow to set namespace as variabile

### CODE REVIEW

File: specs/kubernetes/exec_shell_inside_pod.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Opens a Bash shell into Kubernetes pod
+command: kubectl -n {{namespace}} exec -it {{pod_name}} -- /bin/sh
+arguments:
+  - name: namespace
+    default_value: default
+    description: The namespace of the pod
+  - name: pod_name
+    description: The pod into which to exec
+
+tags: ["kubectl", "exec", "shell", "bash"]
```
Comment (alokedesai) [inline specs/kubernetes/exec_shell_inside_pod.yaml:10]: nit: in general the Warp team recommends reducing the number of tags and to pick ones that are more descriptive. Maybe just `kubectl` here?

**Review by alokedesai — COMMENTED** [review]
Looks great! Just one small comment about reducing the number of tags

File: specs/kubernetes/exec_shell_inside_pod.yaml (hunk 1 above)
Comment (alokedesai) [reply specs/kubernetes/exec_shell_inside_pod.yaml:10]: @radugalbenu Lmk when this is addressed and ready for another pass!

Comment (radugalbenu) [reply specs/kubernetes/exec_shell_inside_pod.yaml:10]: @alokedesai I have created a new PR with the suggested changes, I messed up with the branches I used.
New PR here: https://github.com/warpdotdev/workflows/pull/139 . thanks!

**Comment by alokedesai** [comment]
Awesome! Went ahead and approved that one, and closing this one out

**Pull request closed without merging**